src/box/txn_limbo.h

```diff
@@ -245,6 +251,37 @@ txn_limbo_is_empty(struct txn_limbo *limbo)
 	return rlist_empty(&limbo->queue);
 }
 
+/**
+ * Check whether the queue size limit is reached.
+ * If the queue is full, we must wait for some of the entries to be written
+ * before continuing to replicate more entries.
+ */
+static inline bool
+txn_limbo_queue_is_full(struct txn_limbo *limbo)
```

Comment:
Nit:
`txn_limbo_queue_{is_full,on_append,on_remove}`-> `txn_limbo_{is_full,on_append,on_remove}`

Publicly we call this structure a "synchronous transaction queue" or "synchro queue", but internally it's just `txn_limbo`.